obs2ioda-v2/src/cxx/netcdf_utils.cc

```
@@ -0,0 +1,63 @@
+#include "netcdf_utils.h"
+#include "netcdf_file.h"
+#include <algorithm>
+#include <iostream>
+
+namespace Obs2Ioda {
+
+    std::shared_ptr<netCDF::NcGroup> getRootGroup(
```

cppcheck (2.10-2)

[style] The function 'getRootGroup' is never used.

(unusedFunction)